books.js

```diff
@@ -77,12 +197,12 @@ forms.addEventListener('submit', (event) => {
 });
 
 function populate() {
```

Comment:
- [ ] OPTIONAL
It is a nice idea to prefer _arrow functions_ over normal functions, so as to keep your _function_ syntax simpler and cleaner.